R/po_scan.R

```diff
@@ -0,0 +1,28 @@
+po_scan <- function(dir = ".", custom_translation_functions = list(), verbose = TRUE, style = c("base", "explicit")) {
```

Comment:
I quite like the idea of this "style" argument which allows you to specify which style of translation you're using for your package.

If you like this general idea, we could record it in the DESCRIPTION using something like `Config/potools/style: explicit` so that you could record it once and not have to remember to supply it every time you call `po_scan()`

Reply:
Nice! what's your thinking re: using a package-level config like `.poconfig` vs adding fields to DESCRIPTION? I guess the former makes more sense if there's likely to be more than a small # of possible configuration parameters for the package? e.g. we could record the custom_translation_functions in such a config.

Reply:
Right, if there's only a few settings I think DESCRIPTION is easier since you don't have to add a new file and .Rbuildignore it. 

I'm working on the assumption that hopefully few people will actually need `custom_translation_functions` because potools will just bundle the most common styles.

Reply:
SGTM